test/linalg/lu.jl

```diff
@@ -0,0 +1 @@
+@test logdet(Complex64[1.0f0 0.5f0; 0.5f0 -1.0f0]) === 0.22314355f0 + 3.1415927f0im
```

Comment:
It is in line with @andreasnoack's work in reorganizing the linalg tests by source file. `lu.jl` simply hadn't been done yet.